index.html

```diff
@@ -1,16 +1,118 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Great Idea - Responsive I</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link
+      href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/index.css" />
+  </head>
 
-</head>
+  <body>
+    <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+    <div class="container">
+      <div class="header">
+        <nav>
+          <a href="./services.html" class="first-anchor">Services</a>
+          <a href="#">Product</a>
+          <a href="./vision.html">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </nav>
+        <div class="logo">
+          <img src="img/logo.png" alt="Great Idea! Company logo." />
+        </div>
+      </div>
```

Comment:
Remember your semantic html, this should be a <header> component and the below should be <section>